README.md

```diff
@@ -205,6 +220,61 @@ If tooltips or other overlay components don't appear correctly, ensure you have
 
 If you encounter errors about missing router context when using Link components or other routing-related components, make sure you have wrapped your app with a Router provider from `react-router-dom` (e.g., `BrowserRouter`, `HashRouter`, or `MemoryRouter`).
 
+## Content Security Policy
+
+Echoes React supports strict Content Security Policy (CSP) configurations. To use Echoes with a CSP that restricts inline styles, you need to configure nonces.
+
+### CSP Configuration
+
+Configure your CSP header as follows:
+
+```http
+Content-Security-Policy:
+  style-src
+    'self'
+    'nonce-RANDOM_VALUE'
+    'unsafe-hashes'
+    'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='
+    'sha256-qixoDh78J8vISHKC3rLI7qSXmTShr8mhsUgjJL7W7aU='
+    'sha256-3gJFr3n77fnX5qwQpGju/zCOsoHW5RMqQd5XOb9WFcA=';
```

Comment:
did you try this solution instead: https://github.com/emilkowalski/sonner/issues/449#issuecomment-3017020644 ?